<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FahrzeugSzenarioGUI</class>
 <widget class="QDialog" name="FahrzeugSzenarioGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>178</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fahrzeugfluss verändern...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,5,1">
   <item>
    <widget class="QLabel" name="fahrzeugeProMinuteLabel">
     <property name="text">
      <string>Anzahl von Fahrzeugen (pro Minute):</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fahrzeugeProMinuteLayout">
     <item>
      <widget class="QSpinBox" name="fahrzeugeProMinuteSpinBox">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="wahrscheinlichkeitenLabel">
     <property name="text">
      <string>Wahrscheinlichkeiten für die Abbiegerichtungen:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="1,6,1">
     <item row="0" column="2">
      <widget class="QLabel" name="linksProzentLabel">
       <property name="text">
        <string>in %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="linksLabel">
       <property name="text">
        <string>Links:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="geradeausSlider">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="linksSlider">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="rechtsSlider">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="geradeasLabel">
       <property name="text">
        <string>Geradeaus:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="rechtsLabel">
       <property name="text">
        <string>Rechts:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="geradeausProzentLabel">
       <property name="text">
        <string>in %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="rechtsProzentLabel">
       <property name="text">
        <string>in %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="abbrechenButton">
       <property name="text">
        <string>Abbrechen</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
